From d6cd3fc3aab5ff0bdb3938f0eb05596e324fa089 Mon Sep 17 00:00:00 2001
From: Khang D Nguyen <khangng@os.amperecomputing.com>
Date: Tue, 5 Aug 2025 16:18:31 +0700
Subject: [PATCH] dbus-sdr: remove unused boost process header

Boost process v2 requires linking the module:

    ERROR opening IPMI provider
    /usr/lib/ipmid-providers/libipmi20.so.0.1:
    /usr/lib/ipmid-providers/libipmi20.so.0.1: undefined symbol:
    _ZN5boost7process2v25error22get_exit_code_categoryEv

Seems like we are not even using this header, remove it for good.

Upstream-Status: Submitted [https://gerrit.openbmc.org/c/openbmc/phosphor-host-ipmid/+/82543]

Change-Id: I6a6a6964075514fc0acffdbe79653d0cd2f4607f
Signed-off-by: Khang D Nguyen <khangng@os.amperecomputing.com>
---
 dbus-sdr/storagecommands.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/dbus-sdr/storagecommands.cpp b/dbus-sdr/storagecommands.cpp
index a40b408..9ee3b7f 100644
--- a/dbus-sdr/storagecommands.cpp
+++ b/dbus-sdr/storagecommands.cpp
@@ -22,7 +22,6 @@
 #include <boost/algorithm/string.hpp>
 #include <boost/asio/detached.hpp>
 #include <boost/container/flat_map.hpp>
-#include <boost/process.hpp>
 #include <ipmid/api.hpp>
 #include <ipmid/message.hpp>
 #include <ipmid/types.hpp>
